apps/api/src/app/events/e2e/bridge-trigger.e2e.ts

```diff
@@ -583,7 +583,7 @@ contexts.forEach((context: Context) => {
               controlSchema: {
                 type: 'object',
                 properties: {
-                  name: { type: 'string', default: 'Hello {{name}}' },
+                  name: { type: 'string', default: 'Hello {{payload.name}}' },
```

Comment:
Could we duplicate this file and add a test with the previous `{{name}}` syntax to verify backwards compatibility?